src/PluginForm/LinkTypePlus/LinkTypePlusOffsiteForm.php

```diff
@@ -147,28 +147,25 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
    * @throws \ClientException
    */
   public function getRedirectUrl($order, array $configPayload) {
-    $orderId = $order->id() . '-tt' . $order->getVersion();
+    // $orderId = $order->id() . $order->getVersion();
     $amount = round((string) $order->getBalance()->getNumber());
     $callBackUrlObj = Url::fromUri('route:commerce_gmo_linktypeplus.complete_response');
     $callBackUrlObj->setAbsolute();
     $callBackUrl    = $callBackUrlObj->toString();
-    $pay_methods    = $configPayload['pay_methods'];
-    $cancel_url     = $configPayload['cancel_url'];
-    $return_url     = $configPayload['return_url'];
-
+   
     $this->credentials = [...$this->credentials,'TemplateNo' => $configPayload['template_no']];
 
     $payload['configid'] = $order->id();
     $payload['transaction'] = [
-      'OrderID' => "$orderId",
-      'Amount'  => "$amount",
+      'OrderID' => $order->id(),
```

Comment:
Append $order->getVersion() while we are sending orderID to gmo since it is unique. we are sending order->id.'-'.$order->getVersion()